install/image-references

```diff
@@ -27,6 +27,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:kube-client-agent
+  - name: cluster-etcd-operator
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:cluster-etcd-operator
   - name: cli
```

Comment:
this is not necessary for an operator image, we are referenced in the release by CVO. So go ahead and remove this.